QUARKUS-715.md

```diff
@@ -0,0 +1,55 @@
+# QUARKUS-715 Support for multiple Hibernate ORM persistence units
+
+This document describes a high-level testing overview of [QUARKUS-715 Support for multiple Hibernate ORM persistence units](https://issues.redhat.com/browse/QUARKUS-715).
+
+## Scope of the testing
+
+### Component testing
+The feature consists of boilerplate code allowing the Quarkus applications to hold more than one Hibernate persistence 
+unit in their context.
+
+Hibernate functionality is tested on [Hibernate CI](https://ci.hibernate.org/).
+
+### Integration testing
+Basic set of integration tests has been implemented and merged with the [feature implementation](https://github.com/quarkusio/quarkus/pull/11322/files).
+These tests are executed in Quarkus CI on upstream on a subset of supported platforms.
+
+An integration test verifying correctness of the feature on OpenShift clusters will be implemented in
+[quarkus-qe/quarkus-openshift-test-suite](https://github.com/quarkus-qe/quarkus-openshift-test-suite). The test will be
+an extension of existing SQL test cases to use application with multiple Hibernate persistence units.
+
+[quarkus-qe/beefy-scenarios `004-quarkus-HHH-and-HV`](https://github.com/quarkus-qe/beefy-scenarios/tree/master/004-quarkus-HHH-and-HV) module will also be extended with an application using multiple
+Hibernate persistence units.
+
+### Platform testing
```

Comment:
Maybe we could have this section as a shared page for all test plans?

Reply:
This is changing over time and also not applicable to all RFEs.

Michal could just go with "Tests are going to be executed against all tested versions of databases, RHEL and OpenShift versions. Both JVM and Native mode is gonna be tested."

Reply:
The issue here is we don't have a tested configurations document yet. In EAP, we had document that was updated when each new release was planned/scoped and test plans referred to that.

I've updated the section in accordance with Rosťa's suggestion :)